packages/yoga/src/Divider/native/__snapshots__/Divider.test.jsx.snap

```diff
@@ -916,6 +916,22 @@ exports[`<Divider /> should create a divider component 1`] = `
                   "mobile": 16,
                 },
               },
+              "heading": Object {
+                "paddingBottom": 24,
+                "paddingLeft": 20,
+                "paddingRight": 20,
+                "paddingTop": 20,
+                "rightIcons": Object {
+                  "paddingLeft": 24,
+                },
+                "titleContainer": Object {
+                  "avatar": Object {
+                    "largeSize": 72,
+                    "smallSize": 40,
+                  },
+                  "marginTop": 20,
+                },
+              },
```

Comment:
Why was the Divider's snapshot altered? Were there any changes to this component due to the addition of the new Heading component?

Reply:
I found the problem, in Divider component it's passing all props to styled component, so everything new added will broke the test, I will fix it.

Reply:
Talked via Slack, it is a separate issue to be discussed.